### PR TITLE
feat: separate nwc connection page

### PR DESCRIPTION
**Pull request by rolznz**
Fixes https://github.com/getAlby/bitcoin-connect/issues/9

image

Loading state uses standard skeleton loader from the start page

### CODE REVIEW

File: src/components/pages/bc-nwc.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  }
+  private async onClick() {
+    if (!this._nwcUrl) {
+      // TODO: support toasts?
+      alert('Please enter a URL');
+      return;
+    }
+
+    this._isLoading = true;
+    await store.getState().connect({
```
Comment (rolznz) [inline src/components/pages/bc-nwc.ts:68]: should this return a result or error?

File: src/components/pages/bc-nwc.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    </div>`;
+  }
+
+  private nwcUrlChanged(event: {target: HTMLInputElement}) {
+    this._nwcUrl = event.target.value;
+  }
+  private async onClick() {
+    if (!this._nwcUrl) {
+      // TODO: support toasts?
+      alert('Please enter a URL');
```
Comment (rolznz) [inline src/components/pages/bc-nwc.ts:63]: how would toasts look like here? I don't think we want to add a dependency

Comment (bumi) [reply src/components/pages/bc-nwc.ts:54]: maybe just an error message that is then shown? 
this page could take care of it.

File: src/components/pages/bc-nwc.ts (hunk 1 above)
Comment (bumi) [reply src/components/pages/bc-nwc.ts:59]: if it fails shouldn't we stay on the page that people can fix it?

Comment (rolznz) [reply src/components/pages/bc-nwc.ts:61]: I will address it in https://github.com/getAlby/bitcoin-connect/issues/83

File: src/components/pages/bc-nwc.ts (hunk 2 above)
Comment (rolznz) [reply src/components/pages/bc-nwc.ts:54]: I will address it in https://github.com/getAlby/bitcoin-connect/issues/83

**Pull request merged**